src/index.js

```diff
@@ -5,7 +5,14 @@ const setImmediate = require('async/setImmediate')
 
 module.exports = Block
 
-// Immutable block of data
+/**
+ * Represents an immutable block of data that is uniquely referenced with a multihash key.
+ *
+ * @constructor
+ * @param {Buffer | string} data - The data to be stored in the block as a buffer or a UTF8 string.
+ * @example
+ * var block = new Block('a012d83b20f9371...');
```

Comment:
Please use the same style for examples we use in the rest of the code, (no semicolons, prefer `const` and `let` over `var`)